subprojects/gradle/runner/device-provider/build.gradle.kts

```diff
@@ -0,0 +1,43 @@
+plugins {
+    id("kotlin")
+    `maven-publish`
+    id("com.jfrog.bintray")
+    `java-test-fixtures`
+    id("nebula.integtest")
+}
+
+extra["artifact-id"] = "runner-device-provider"
+
+dependencies {
+    // todo api impl?
+    api(project(":gradle:runner:service"))
+    api(project(":gradle:runner:shared"))
+    api(project(":gradle:runner:stub"))
+    api(project(":gradle:kubernetes"))
+
+    implementation(Dependencies.funktionaleTry)
+    implementation(project(":gradle:process"))
+    implementation(project(":common:logger"))
+
+    testImplementation(project(":common:logger-test-fixtures"))
+}
+
+kotlin {
+    explicitApi()
+
+    /**
+     * Workaround to access internal classes from textFixtures
+     * till https://youtrack.jetbrains.com/issue/KT-34901 resolved
+     */
```

Comment:
Can we apply it to all modules?

Reply:
yes, will do eventually; wanted to use convention plugin for this